src/main/kotlin/ecommerce/service/MemberService.kt

```diff
@@ -0,0 +1,45 @@
+package ecommerce.service
+
+import ecommerce.dto.member.RegisterRequest
+import ecommerce.model.Member
+import ecommerce.repository.MemberRepository
+import org.springframework.stereotype.Service
+
+@Service
+class MemberService(
+    private val memberRepository: MemberRepository,
+    private val passwordService: PasswordService,
+    private val tokenService: TokenService,
+) {
+    fun register(request: RegisterRequest): String {
+        if (memberRepository.existsByEmail(request.email)) {
+            throw IllegalArgumentException("Email already exists")
+        }
+        val hashedPassword = passwordService.hashPassword(request.password)
```

Comment:
Great Idea! Since `PasswordService` handles critical logic, it would be great to have unit tests for it.